### ISSUE TITLE
UNMAINTAINED Notice

### ISSUE THREAD

**Issue by holybit** [spf13/spf13-vim#982]
From what I can discern this Vim distro doesn't look like it's being maintained anymore. Please, add bold notice to the top of the README that this distro is no longer maintained and possibly a few links to other worthy vim distros.

**Comment by sensorii** [comment]
https://github.com/carlhuda/janus

**Comment by sensorii** [comment]
https://github.com/bling/dotvim

**Comment by holybit** [comment]
@sensorii The last [commit](https://github.com/spf13/spf13-vim/commits/3.0) to master was Feb 13, 2016. Probably time to put an abandoned notification at the top of the repo so folks don't waste their time setting up a dead vim distro.

**Comment by sensorii** [comment]
I agree, @holybit, notification would be nice. Or it could come back to life with a new maintainer, or maybe there's an active fork out there. I'm not a maintainer of this project. I'm looking at switching to [Janus](https://github.com/carlhuda/janus) as it is alive... now if I could just get parts of Janus to resemble my spf13-vim setup that I like so much...

**Comment by dessalines** [comment]
Yeah, this distro is dead. Hasn't been an update in over a year.

**Comment by ahnwooseon** [comment]
Eight years.
